docs/notebooks/binning_slowly_changing_sources.ipynb

```diff
@@ -0,0 +1,369 @@
+{
```

Comment:
<p>I wonder if sentence The <code>offset</code> parameter allows the user to indicate where the division point should be between days'' can be clearer. I think the confusion arrises because we use a word day do mean both 24 hour period and the bright part of the that period (i.e., opposite to night). In this case these meanings are hard to distinguish. Something more like - The <code>offset</code> parameter allows the user to indicate where the division between different nights/observing blocks should be. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lincc-frameworks/lsstseries/pull/96/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
I like your phrasing better and used that.